corehq/apps/userreports/sql/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .adapter import get_indicator_table, IndicatorSqlAdapter, metadata, ErrorRaisingIndicatorSqlAdapter
 from .columns import get_expanded_column_config, SqlColumnConfig
-from .util import get_column_name, get_table_name, truncate_value
+from .util import get_column_name
```

Comment:
It is actually used at the moment (but not in this file). some places call 

`from corehq.apps.userreports.sql import get_column_name`